src/ert/ensemble_evaluator/identifiers.py

```diff
@@ -17,28 +17,28 @@
 STDOUT = "stdout"
 STEPS = "steps"
 
-EVTYPE_FM_STEP_FAILURE = "com.equinor.ert.forward_model_step.failure"
-EVTYPE_FM_STEP_PENDING = "com.equinor.ert.forward_model_step.pending"
-EVTYPE_FM_STEP_RUNNING = "com.equinor.ert.forward_model_step.running"
-EVTYPE_FM_STEP_SUCCESS = "com.equinor.ert.forward_model_step.success"
-EVTYPE_FM_STEP_UNKNOWN = "com.equinor.ert.forward_model_step.unknown"
-EVTYPE_FM_STEP_WAITING = "com.equinor.ert.forward_model_step.waiting"
-EVTYPE_FM_STEP_TIMEOUT = "com.equinor.ert.forward_model_step.timeout"
+EVTYPE_REALIZATION_FAILURE = "com.equinor.ert.realization.failure"
```

Comment:
These are the new realization statuses.

Reply:
I think I will call these *renamed* events.  There were realization *states* before, but they were implicitly updated based on the step states by the function `_propagate_step_update_to_realization`. Now we update them directly using their own events.